src/screens/VisualTests/BuildResults.tsx

```diff
@@ -129,6 +134,27 @@ export const BuildResults = ({
     <Sections>
       {buildStatus}
 
+      {!buildStatus &&
+        (!canReview || !isReviewable) &&
+        storyBuild.status === BuildStatus.Pending && (
```

Comment:
I prefer pulling complex logic out into vars which kind of explains what's happening here.